PROTO_tests/tests/alignment_test.py

```diff
@@ -0,0 +1,167 @@
+import pytest
+import arkouda as ak
+
+DATA_TYPES = [ak.int64, ak.uint64, ak.float64]
+
+
+class TestAlignment:
+    @staticmethod
+    def get_interval_info(lower_bound, upper_bound, vals, dtype):
+        if dtype == ak.uint64:
+            lower_bound = [i + 2**63 for i in lower_bound]
+            upper_bound = [i + 2**63 for i in upper_bound]
+            vals = [i + 2**63 for i in vals]
+        elif dtype == ak.float64:
+            lower_bound = [i + 0.5 for i in lower_bound]
+            upper_bound = [i + 0.5 for i in upper_bound]
+            vals = [i + 0.5 for i in vals]
+
+        lb = ak.array(lower_bound, dtype)
+        ub = ak.array(upper_bound, dtype)
+        v = ak.array(vals, dtype)
+
+        return lb, ub, v
```

Comment:
i feel this might be cleaner if you create the arrays and then shift. This is purely stylistic so don't feel obligated to change
```python
lb = ak.array(lower_bound, dtype)
ub = ak.array(upper_bound, dtype)
v = ak.array(vals, dtype)

if dtype == ak.uint64:
    lb += 2**63
    ub += 2**63
    v += 2**63
elif dtype == ak.float64:
    lb += 0.5
    ub += 0.5
    v += 0.5
return lb, ub, v
```

Reply:
Updated